config/imsim-config.yaml

```diff
@@ -66,6 +66,10 @@ input:
         visit: 19500
         snap: 1
 
+    camera_data:
```

Comment:
Sadly, the changes in this file are also going to need to go into the two new template files with the copy.

(can handle after that PR merges to main..  just noting now)